apps/website/components/templates/search/search.tsx

```diff
@@ -32,7 +32,7 @@ export function SearchTemplate({ query }: TemplateProps) {
   const { activeTab, handleTabChange } = useTab();
 
   const { data, isLoading } = useQuery<SearchQuery>(
-    `search-${query}`,
+    [`search-${query}`],
```

Comment:
```suggestion
    ['search', query],
```

It's prettier and better for querying cached data I would say.

Reply:
it actually does the same thing

Reply:
Yeah I know, it was more of a suggestion than an actual bug fix